ee/tabby-ui/app/auth/signin/components/user-signin-form.tsx

```diff
@@ -42,15 +45,37 @@ interface UserAuthFormProps extends React.HTMLAttributes<HTMLDivElement> {
   invitationCode?: string
 }
 
+const DEMO_PROFILE = {
+  EMAIL: 'demo@tabbyml.com',
+  PASSWORD: '0$TabbyDemo'
+}
+
 export default function UserSignInForm({
   className,
   invitationCode,
   ...props
 }: UserAuthFormProps) {
   const isEmailConfigured = useIsEmailConfigured()
+  const isDemoMode = useIsDemoMode()
   const form = useForm<z.infer<typeof formSchema>>({
     resolver: zodResolver(formSchema)
   })
+  const formRef = React.useRef<HTMLFormElement | null>(null)
+
+  React.useEffect(() => {
+    const storageKey = 'isAutoLoggedIn'
```

Comment:
```suggestion
    const storageKey = '_tabby_demo_autologin'
```

also put this as a global var

Reply:
Added into `lib/constants/index.ts`